src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -57,7 +57,7 @@
   private SimpleMotorFeedforward simpleMotorFF;
 
   private final DifferentialDriveOdometry odometry;
-  private AHRS gyro;
+  public static AHRS gyro;
```

Comment:
I don't understand how this fits with the changes, or why it's needed at all.  Doesn't seem like gyro is referenced outside of drivebasesybsystem.

Reply:
The gyro is necessary for some sequences of the automatic climb.